guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -48,11 +48,21 @@ If an attacker, located in the network between {Project} and a host, fetches the
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
 
-. From the *Remote Execution* list, select whether you want to deploy {Project} SSH keys to hosts or not.
+. From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
++
+If set to `Yes`, SSH keys will be installed on the registered host. The inherited value is based on the `host_registration_remote_execution` parameter.
+It can be inherited e.g. from host group, operating system, organization. When overridden, the selected value will be stored on host parameter level.
++
 
-ifdef::satellite[]
-. From the *Insights* list, select whether you want to register the hosts to Insights or not.
-endif::[]
+. From the *Setup Insights* list, select whether you want to install `insights-client` and register the hosts to Insights.
++
+Insights tool is available for Red Hat family operating systems only, it has no effect on other OS families that do not support it.
```

Comment:
AFAIK it's only available on RHEL, not on the Red Hat family. At least CentOS Stream 8 and CentOS Linux 7 don't have it.
```suggestion
The Insights tool is available for Red Hat Enterprise Linux only. It has no effect on other operating systems.
```

Reply:
I shamelessly copy/pasted the text from label help in registration form, guess we should change it there too (and the API too) ...